setup.cfg

```diff
@@ -2,15 +2,15 @@
 python_files = test*.py
 # Note that the database is reused on tests, so run `pytest --create-db` before running the tests
 # if your models have changed
-addopts = --reuse-db --tb=native -p no:doctest -p no:warnings -s
```

Comment:
What does removing the `-s` do?

Reply:
The `-s` flag was added by me by accident. It turns on a bunch of extra logging on success (turns off "capturing" logs).